cron/centreon-backup.pl

```diff
@@ -312,12 +312,12 @@ ()
 	my @tab_php_ini;
 
 	# PHP CLI
-	my $result = `php -r 'echo php_ini_loaded_file();'`;
+	my $result = `/opt/rh/rh-php71/root/usr/bin/php -r 'echo php_ini_loaded_file();'`;
```

Comment:
Please use the @PHP_BIN@ macro that will be replaced automatically during packaging.

Reply:
Fixed in next commit